MultimodalQnA/docker_compose/intel/cpu/xeon/set_env.sh

```diff
@@ -6,30 +6,50 @@ pushd "../../../../../" > /dev/null
-export http_proxy=${your_http_proxy}
-export https_proxy=${your_http_proxy}
-export EMBEDDER_PORT=6006
-export MMEI_EMBEDDING_ENDPOINT="http://${host_ip}:$EMBEDDER_PORT/v1/encode"
-export MM_EMBEDDING_PORT_MICROSERVICE=6000
-export ASR_ENDPOINT=http://$host_ip:7066
+export host_ip=$(hostname -I | awk '{print $1}')
+export MM_EMBEDDING_SERVICE_HOST_IP=${host_ip}
+export MM_RETRIEVER_SERVICE_HOST_IP=${host_ip}
+export LVM_SERVICE_HOST_IP=${host_ip}
+export MEGA_SERVICE_HOST_IP=${host_ip}
+
+export no_proxy=${no_proxy}
+export http_proxy=${http_proxy}
+export https_proxy=${http_proxy}
+
+export WHISPER_PORT=7066
+export WHISPER_MODEL="base"
+export ASR_ENDPOINT=http://$host_ip:$WHISPER_PORT
+
+export ASR_PORT=9099
 export ASR_SERVICE_PORT=3001
 export ASR_SERVICE_ENDPOINT="http://${host_ip}:${ASR_SERVICE_PORT}/v1/audio/transcriptions"
-export REDIS_URL="redis://${host_ip}:6379"
+
+export REDIS_DB_PORT=6379
+export REDIS_INSIGHTS_PORT=8001
+export REDIS_URL="redis://${host_ip}:${REDIS_DB_PORT}"
 export REDIS_HOST=${host_ip}
 export INDEX_NAME="mm-rag-redis"
-export LLAVA_SERVER_PORT=8399
-export LVM_ENDPOINT="http://${host_ip}:8399"
+
+export DATAPREP_MMR_PORT=6007
+export DATAPREP_INGEST_SERVICE_ENDPOINT="http://${host_ip}:${DATRAPREP_MMR_PORT}/v1/ingest_with_text"
+export DATAPREP_GEN_TRANSCRIPT_SERVICE_ENDPOINT="http://${host_ip}:${DATRAPREP_MMR_PORT}/v1/generate_transcripts"
+export DATAPREP_GEN_CAPTION_SERVICE_ENDPOINT="http://${host_ip}:${DATRAPREP_MMR_PORT}/v1/generate_captions"
+export DATAPREP_GET_FILE_ENDPOINT="http://${host_ip}:${DATRAPREP_MMR_PORT}/v1/dataprep/get_files"
+export DATAPREP_DELETE_FILE_ENDPOINT="http://${host_ip}:${DATRAPREP_MMR_PORT}/v1/dataprep/delete_files"
+
+export EMM_BRIDGETOWER_PORT=6006
 export EMBEDDING_MODEL_ID="BridgeTower/bridgetower-large-itm-mlm-itc"
+export MMEI_EMBEDDING_ENDPOINT="http://${host_ip}:$EMM_BRIDGETOWER_PORT/v1/encode"
+export MM_EMBEDDING_PORT_MICROSERVICE=6000
+
+export REDIS_RETREIEVER_PORT=7000
+
+export LVM_PORT=9399
+export LLAVA_SERVER_PORT=8399
 export LVM_MODEL_ID="llava-hf/llava-1.5-7b-hf"
-export WHISPER_MODEL="base"
-export MM_EMBEDDING_SERVICE_HOST_IP=${host_ip}
-export MM_RETRIEVER_SERVICE_HOST_IP=${host_ip}
-export LVM_SERVICE_HOST_IP=${host_ip}
-export MEGA_SERVICE_HOST_IP=${host_ip}
-export BACKEND_SERVICE_ENDPOINT="http://${host_ip}:8888/v1/multimodalqna"
-export DATAPREP_INGEST_SERVICE_ENDPOINT="http://${host_ip}:6007/v1/ingest_with_text"
-export DATAPREP_GEN_TRANSCRIPT_SERVICE_ENDPOINT="http://${host_ip}:6007/v1/generate_transcripts"
-export DATAPREP_GEN_CAPTION_SERVICE_ENDPOINT="http://${host_ip}:6007/v1/generate_captions"
-export DATAPREP_GET_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/get_files"
-export DATAPREP_DELETE_FILE_ENDPOINT="http://${host_ip}:6007/v1/dataprep/delete_files"
+export LVM_ENDPOINT="http://${host_ip}:${LLAVA_SERVER_PORT}"
+
+export MEGASERVICE_PORT=8888
+export BACKEND_SERVICE_ENDPOINT="http://${host_ip}:${MEGASERVICE_PORT}/v1/multimodalqna"
```

Comment:
Should this use `${MEGA_SERVICE_HOST_IP}` instead of `${host_ip}`? I don't really know why there are those *_HOST_IP env vars, but they aren't always used. It looks like that's how it already was before your changes though, right?

Reply:
Yes it's redundant. But didn't change it at the time because I wasn't sure of it was being used somewhere